Scenes/Map/Block.gd

```diff
@@ -9,13 +9,11 @@ var rng = RandomNumberGenerator.new()
 
 func explode():
 	call_deferred("queue_free")
-	var random = rng.randi_range(0, 2)
-	if (random < 2):
+	var random = rng.randi_range(0, 4)
+	if (random < 3):
```

Comment:
Chyba powinniśmy to kiedyś zmienić na zmienną, a najlepiej na wielkość tablicy po prostu